src/app/features/preprints/services/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './brand.service';
```

Comment:
Use here exact naming for service instead of `*`.
```suggestion
export { BrandService } from './brand.service';
```